locations/spiders/liberty_au.py

```diff
@@ -0,0 +1,31 @@
+import scrapy
+
+from locations.items import Feature
+
+
+class LibertyAUSpider(scrapy.Spider):
+    name = "liberty_au"
+    item_attributes = {"brand": "Liberty Oil", "brand_wikidata": "Q106687078"}
+    allowed_domains = ["libertyconvenience.com.au"]
+    start_urls = ["https://www.libertyconvenience.com.au/wp-admin/admin-ajax.php?action=store_search&autoload=1"]
+
+    def parse_hours(store_hours):
+        pass
```

Comment:
This could be:

```
from scrapy import Select

from locations.hours import OpeningHours

hours_string = " ".join(filter(None, Select(text=location["hours"]).xpath('//text()').getall()))
item["opening_hours"] = OpeningHours()
item["opening_hours"].add_ranges_from_string(hours_string)
```